common/models/service-instance.json

```diff
@@ -24,7 +24,12 @@
     },
     "startTime": {
       "type": "date",
-      "description": "The ID of the service group this executor belongs to"
+      "description": "Timestamp when instance was started"
+    },
+    "stopTime": {
+      "type": "date",
+      "description": "Timestamp when instance was stopped (cloud usecases)",
+      "default": "1/1/1970 00:00:00 GMT+0"
```

Comment:
Shouldn't this default to `null` if it is going to be doubling as a state flag?

(sorry for the double comment, the first one was on the commit not the PR diff)